src/validators/twitter-username-validator.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Yoast\WP\SEO\Validators;
+
+use Yoast\WP\SEO\Exceptions\Validation\Abstract_Validation_Exception;
+use Yoast\WP\SEO\Exceptions\Validation\Invalid_Twitter_Username_Exception;
+
+/**
+ * The Twitter username validator class.
+ */
+class Twitter_Username_Validator extends Text_Field_Validator {
+
+	// phpcs:disable Squiz.Commenting.FunctionCommentThrowTag.WrongNumber -- Reason: The parent validate can throw too.
```

Comment:
This is applicable only to the `validate` function, lets enable it after that function again? (even though there is not much else to this class right now)